playground/frontend/lib/l10n/app_en.arb

```diff
@@ -0,0 +1,70 @@
+{
```

Comment:
This file may be what is causing the rat task failure.  You can run the rat task on your machine:
```
cd /path/to/apache/beam/on/your/machine
./gradlew rat
```

Reply:
Hello @damondouglas, thank you for reviewing , yes, i think i forgot to add *.arb files to analysis exclusions, i'll fix this issue in 5 minutes